docs/en/query_language/functions/other_functions.md

```diff
@@ -6,12 +6,14 @@ Returns a string with the name of the host that this function was performed on.
 
 ## basename
 
-Extracts trailing part of a string after the last slash or backslash.
+Extracts trailing part of a string after the last slash or backslash. This function if often used to extract the filename from the path.
 
 ```
 basename( expr )
 ```
 
+Backslashes should be escaped in `expr`.
```

Comment:
* sounds misleading: not in expression itself, but in string result it produces
* maybe it's better to move this to Parameters section below